server/service/modules/activities/activities.go

```diff
@@ -0,0 +1,114 @@
+package activities
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/cenkalti/backoff"
+	"github.com/fleetdm/fleet/v4/server"
+	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	kithttp "github.com/go-kit/kit/transport/http"
+	kitlog "github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+)
+
+type activityModule struct {
```

Comment:
Nit on naming, "modules" are a thing in Go and mean something different, I'm afraid this could become confusing. (https://go.dev/blog/using-go-modules)